### PR TITLE
[5.2] Created translatable strings for LocalAdapter.php "code review"

### PR DESCRIPTION
**Pull request by webmasterab**
Pull Request for Issue # .

This is new to me.
If I did it wrong please let me know.

Summary of Changes
I changed fixed translation to translatable strings

Testing Instructions

"code review"
It's only changed in the code.
I converted the fixed value of the error message into a translatable string

Actual result BEFORE applying this Pull Request
In localAdapter.php there was untranslatable text.

Expected result AFTER applying this Pull Request
In localAdapter.php the text is now translatable using string

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [X ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ X] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: plugins/filesystem/local/src/Adapter/LocalAdapter.php [hunk 1]
```diff
@@ -537,15 +537,16 @@ private function copyFile(string $sourcePath, string $destinationPath, bool $for
...
         if (is_file($destinationPath) && !File::delete($destinationPath)) {
-            throw new \Exception('Copy folder is not possible as destination folder is a file and can not be deleted');
+            throw new \Exception(Text::_('COM_MEDIA_COPY_FOLDER_DESTINATION_CAN_NOT_DELETE'));
         }
 
         if (!Folder::copy($sourcePath, $destinationPath, '', $force)) {
-            throw new \Exception('Copy folder is not possible');
+            throw new \Exception(Text::_('COM_MEDIA_COPY_FOLDER_NOT_POSSIBLE'));
+            
         }
```
Comment (Quy) [inline plugins/filesystem/local/src/Adapter/LocalAdapter.php]:
```suggestion
            throw new \Exception(Text::_('COM_MEDIA_COPY_FOLDER_NOT_POSSIBLE'));
        }
```

File: language/en-GB/com_media.ini [hunk 2]
```diff
@@ -31,6 +36,7 @@ COM_MEDIA_CURRENT_PROGRESS="Current progress"
 COM_MEDIA_DECREASE_GRID="Decrease grid size"
 COM_MEDIA_DELETE_ERROR="Error deleting the item."
 COM_MEDIA_DELETE_SUCCESS="Item deleted."
+COM_MEDIA_DELETE_NOT_POSSIBLE="Delete not possible!"
```
Comment (Quy) [inline language/en-GB/com_media.ini]:
```suggestion
COM_MEDIA_DELETE_ERROR="Error deleting the item."
COM_MEDIA_DELETE_NOT_POSSIBLE="Delete not possible!"
COM_MEDIA_DELETE_SUCCESS="Item deleted."
```

**Comment by toivo** [comment]
I have tested this item :red_circle: unsuccessfully on a07d5c2d7913b39f32ad6328aa9396e9a7c052b8<br /><br />Tested briefly in Joomla 4.4.6-dev. The language strings should be added also to administrator/language/en-GB/com_media.ini. Value: COM_MEDIA_MOVE_FILE_NOT_POSSIBLE="COM_MEDIA_MOVE_FILE_NOT_POSSIBLE"<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43694">issues.joomla.org/tracker/joomla-cms/43694</a>.</sub>

**Comment by webmasterab** [comment]
> I have tested this item 🔴 unsuccessfully on [a07d5c2](https://github.com/joomla/joomla-cms/commit/a07d5c2d7913b39f32ad6328aa9396e9a7c052b8)Tested briefly in Joomla 4.4.6-dev. The language strings should be added also to administrator/language/en-GB/com_media.ini. Value: COM_MEDIA_MOVE_FILE_NOT_POSSIBLE="COM_MEDIA_MOVE_FILE_NOT_POSSIBLE"
> 
> This comment was created with the [J!Tracker Application](https://github.com/joomla/jissues) at [issues.joomla.org/tracker/joomla-cms/43694](https://issues.joomla.org/tracker/joomla-cms/43694).

Thank you for your test.
Too bad this didn't work out.

I understand why you marked it as unsuccessful.

How can I update this request so that I can also put the missing translations in the admin's.
Or should I make another request for this?

**Comment by webmasterab** [comment]
@Quy and @toivo I have adjusted it and now also adjusted the file in administrator/language.
Would you please look and possibly do a check.
Thanks

**Comment by toivo** [comment]
I have tested this item :white_check_mark: successfully on 0b36e42105a697582e2e950fb2dce93c9bcaeea5<br /><br />Tested successfully in Joomla 4.4.6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43694">issues.joomla.org/tracker/joomla-cms/43694</a>.</sub>

File: administrator/language/en-GB/com_media.ini [hunk 3]
```diff
@@ -82,6 +88,12 @@ COM_MEDIA_MEDIA_MIME_TYPE="MIME Type"
 COM_MEDIA_MEDIA_NAME="Name"
 COM_MEDIA_MEDIA_SIZE="Size"
 COM_MEDIA_MEDIA_TYPE="Type"
+COM_MEDIA_MOVE_FILE_ALREADY_EXISTS="Move file is not possible as destination file already exists"
+COM_MEDIA_MOVE_FILE_EXTENSION_INVALID="Move file is not possible as the extension is invalid"
+COM_MEDIA_MOVE_FILE_NOT_POSSIBLE="COM_MEDIA_MOVE_FILE_NOT_POSSIBLE"
```
Comment (Quy) [inline administrator/language/en-GB/com_media.ini]: Please fix.

Comment (webmasterab) [reply administrator/language/en-GB/com_media.ini]: > Please fix.

I didn't see thanks.
It has now been adjusted

Comment (Quy) [reply administrator/language/en-GB/com_media.ini]: It should be `Move file is not possible`. Thanks.

Comment (webmasterab) [reply administrator/language/en-GB/com_media.ini]: @Quy Is it good now or should it be named differently?
I got it from the hardcoded version of the php file.
See the modified files

Comment (Quy) [reply administrator/language/en-GB/com_media.ini]: No it is still wrong. See screenshot. Please compare the language string to the original text.

Comment (webmasterab) [reply administrator/language/en-GB/com_media.ini]: I indeed had the wrong text.
It really has to be right now.

Sorry and thanks for pointing it out to me.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 23165009384ce77a50b5a7ade1a9d14ffd57ee15<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43694">issues.joomla.org/tracker/joomla-cms/43694</a>.</sub>

**Comment by fgsw** [comment]
I have tested this item :white_check_mark: successfully on 23165009384ce77a50b5a7ade1a9d14ffd57ee15<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43694">issues.joomla.org/tracker/joomla-cms/43694</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43694">issues.joomla.org/tracker/joomla-cms/43694</a>.</sub>

**Comment by webmasterab** [comment]
@Quy Thanks for merging with 4.4.
However, when I look at that code now, I don't see my adjustments, is that right?

**Comment by brianteeman** [comment]
it has not been merged yet

**Comment by brianteeman** [comment]
ah I see the confusion

@quy has merged the latest changes from core into this pull request in preparation for this pull request to be merged into core

**Comment by webmasterab** [comment]
@brianteeman Thank you for explaining.
I saw that it was merged with joomla-dev4.4 so I thought it worked.
I'm waiting

**Comment by laoneo** [comment]
Before I can merge, you need to resolve the conversation. If it is not fixed and you need to make an adjustment with a new commit, then we need another round of testing.

**Comment by webmasterab** [comment]
> Before I can merge, you need to resolve the conversation. If it is not fixed and you need to make an adjustment with a new commit, then we need another round of testing.

I need to close the pull request.

Or what should I do, I don't understand it well.

**Comment by laoneo** [comment]
I'v rebased this to 5.2. As soon as 5.2 gets the upmerges from 4.4 we can update this one too and the conflicts will be gone.

**Comment by Hackwar** [comment]
Thanks for your contribution @webmasterab!

**Pull request merged**